spec/features/user_edit_profile_spec.rb

```diff
@@ -0,0 +1,85 @@
+require 'rails_helper'
+
+RSpec.describe "Users profile page", driver: :selenium_headless, type: :feature do
+  let(:user) { FactoryBot.create :user }
+
+  before do
+    user.social_accounts.create(social_network: "GitHub", user_name: "SomeGitUserName")
+    user.save()
+    user.social_accounts.create(social_network: "Telegram", user_name: "SomeTelegramUserName")
+    user.save()
+    sign_in user
+end
+
+it 'show social account' do
+    visit edit_user_path(user)
+
+    expect(page).to have_text("GitHub")
+    expect(page).to have_text("SomeGitUserName")
+    expect(page).to have_text("Telegram")
+    expect(page).to have_text("SomeTelegramUserName")
+  end
+
+  it 'provides link to edit social account' do
+    visit edit_user_path(user)
+    page.all(:link, 'Edit')[0].click
+    expect(find_field("Social network").value).to eq "GitHub"
+    expect(find_field("User name").value).to eq "SomeGitUserName"
+  end
+
+  it 'provides link to remove social account' do
+    visit edit_user_path(user)
+    
+    page.all(:link, 'Remove')[1].click
```

Comment:
If I would add another link named 'Remove`, this test would probably break because of the magic index